autogpt_platform/frontend/src/components/contextual/PublishAgentModal/components/AgentSelectStep/AgentSelectStep.tsx

```diff
@@ -162,10 +165,42 @@ export function AgentSelectStep({
                         />
                       </div>
                       <div className="flex flex-col gap-2 p-3">
-                        <Text variant="large-medium">{agent.name}</Text>
-                        <Text variant="small" className="!text-neutral-500">
-                          Edited {agent.lastEdited}
+                        <Text variant="large-medium" className="line-clamp-2">
+                          {agent.name}
                         </Text>
+                        <div className="flex items-center justify-between gap-2">
+                          <div className="flex-1">
+                            <Text variant="small" className="!text-neutral-500">
```

Comment:
You should not need `!` to override the color?

Reply:
updated